modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -1546,6 +1551,36 @@ private long checkPoolStarvation(
                 EventType.EVT_NODE_JOINED, localNode());
 
         startTimer.finishGlobalStage("Await exchange");
+
+        if (log.isInfoEnabled())
+            ackExpPlcInfoByCaches();
+    }
+
+    /**
+     * Acks expiration policy info by caches.
+     */
+    private void ackExpPlcInfoByCaches() {
+        final String r = ctx.cache().context().cacheContexts().stream()
+                .map(cacheCtx -> {
+                    ExpiryPolicy expPlc = cacheCtx.expiry();
+                    if (expPlc == null || expPlc instanceof EternalExpiryPolicy) return null;
+
+                    Duration dur;
+                    if (expPlc.getExpiryForCreation() != null)
+                        dur = expPlc.getExpiryForCreation();
+                    else if (expPlc.getExpiryForUpdate() != null)
+                        dur = expPlc.getExpiryForUpdate();
+                    else
+                        dur = expPlc.getExpiryForAccess();
+
+                    if (dur == null || dur.getTimeUnit() == null) return null;
+
+                    return "{cache=" + cacheCtx.name() + ", duration=" + dur.getTimeUnit().toMillis(dur.getDurationAmount()) +
+                            "ms, isEagerTtl=" + cacheCtx.ttl().eagerTtlEnabled() + "}";
+                }).filter(s -> !Objects.isNull(s))
+                .collect(Collectors.joining(", "));
+
+        log.info(String.format("Expiration policy info for caches [%s]", r));
```

Comment:
do we need to print this message if previously return null and thus final String r = "" ?